playbooks/virt/openshift-cluster/meta-data

```diff
@@ -0,0 +1,2 @@
+instance-id: {{ item[0] }}
+local-hostname: {{ item[0] }}
```

Comment:
Could you create a templates directory and put the meta-data, user-data and domain.xml files there?